### ISSUE TITLE
Worldedit 

### ISSUE THREAD

**Issue by tommykent1210** [ColonyPlusPlus/ColonyPlusPlus-mod#25]
Add some worldedit style commands to the game, similar to how this worked in Minecraft.

Client API would be GREAT...

**Comment by DragoWhooves** [comment]
maybe have two versions of the command, one that requires no blocks and a version that requires you to have the relevant blocks in your stockpile (and maybe even dumps any cleared blocks into Stockpile)

**Comment by tommykent1210** [comment]
Yeah, I was planning on basically having a switch that lets you do it in creative

so

`/worldedit rectangle stonebrick` and `/worldedit rectangle stonebrick creative`

**Comment by tommykent1210** [comment]
Proposed commands:
* `/worldedit fill {itemtype}` - fills the space between 2 markers with the type
* `/worldedit hollow {itemtype}` - fills the space between 2 markers with the item, leaving air inside
* `/worldedit sphere {itemtype}` - generate a sphere between 2 markers (probably top left bottom right
* `/worldedit disc/disk {itemtype}` - fills the space between 2 markers with the type, creating a disc/circle (instead of a rectangle)

**Comment by tommykent1210** [comment]
Unlikely to make it into 0.2.0, deferring
